<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Пошук по XML файлу</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/katynko/svg/edit-table-insert-row-below.svg</normaloff>:/katynko/svg/edit-table-insert-row-below.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/katynko/svg/go-home.svg</normaloff>:/katynko/svg/go-home.svg</iconset>
          </attribute>
          <attribute name="title">
           <string>Домівка</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <property name="leftMargin">
            <number>3</number>
           </property>
           <property name="topMargin">
            <number>3</number>
           </property>
           <property name="rightMargin">
            <number>3</number>
           </property>
           <property name="bottomMargin">
            <number>3</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="leftMargin">
              <number>9</number>
             </property>
             <property name="rightMargin">
              <number>9</number>
             </property>
             <item>
              <widget class="QToolButton" name="tbFileHistory">
               <property name="toolTip">
                <string>Раніше відкриті файли</string>
               </property>
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset resource="res.qrc">
                 <normaloff>:/katynko/svg/format-list-ordered.svg</normaloff>:/katynko/svg/format-list-ordered.svg</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="tbPath2file">
               <property name="text">
                <string>Файл</string>
               </property>
               <property name="icon">
                <iconset resource="res.qrc">
                 <normaloff>:/katynko/svg/document-open.svg</normaloff>:/katynko/svg/document-open.svg</iconset>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextBesideIcon</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="leFileName">
               <property name="text">
                <string/>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="tbSettings">
               <property name="text">
                <string>Налаштування</string>
               </property>
               <property name="icon">
                <iconset resource="res.qrc">
                 <normaloff>:/katynko/svg/applications-system.svg</normaloff>:/katynko/svg/applications-system.svg</iconset>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextBesideIcon</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>Особа</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QRadioButton" name="rbUr">
                <property name="text">
                 <string>Юр. особа</string>
                </property>
                <property name="icon">
                 <iconset resource="res.qrc">
                  <normaloff>:/katynko/svg/im-kick-user.svg</normaloff>:/katynko/svg/im-kick-user.svg</iconset>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="rbFop">
                <property name="text">
                 <string>ФОП</string>
                </property>
                <property name="icon">
                 <iconset resource="res.qrc">
                  <normaloff>:/katynko/svg/im-user.svg</normaloff>:/katynko/svg/im-user.svg</iconset>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Фільтр</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QStackedWidget" name="swUrOrFop">
                <property name="currentIndex">
                 <number>0</number>
                </property>
                <widget class="QWidget" name="page_3">
                 <layout class="QGridLayout" name="gridLayout_2">
                  <item row="0" column="2">
                   <widget class="QLineEdit" name="leFltrUr">
                    <property name="placeholderText">
                     <string>Значення для пошуку</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="3">
                   <widget class="QPushButton" name="pbAddFltr">
                    <property name="text">
                     <string>Додати</string>
                    </property>
                    <property name="icon">
                     <iconset resource="res.qrc">
                      <normaloff>:/katynko/svg/draw-cross.svg</normaloff>:/katynko/svg/draw-cross.svg</iconset>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QComboBox" name="cbxFltrUr">
                    <property name="sizeAdjustPolicy">
                     <enum>QComboBox::AdjustToContents</enum>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0">
                   <widget class="QToolButton" name="tbRemoveFltrUr">
                    <property name="toolTip">
                     <string>Видалити з пошуку</string>
                    </property>
                    <property name="text">
                     <string>...</string>
                    </property>
                    <property name="icon">
                     <iconset resource="res.qrc">
                      <normaloff>:/katynko/svg/edit-delete.svg</normaloff>:/katynko/svg/edit-delete.svg</iconset>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0" colspan="4">
                   <widget class="QPlainTextEdit" name="pteFltrUr">
                    <property name="readOnly">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="page_4">
                 <layout class="QGridLayout" name="gridLayout_4">
                  <item row="0" column="1">
                   <widget class="QComboBox" name="cbxFltrFop">
                    <property name="sizeAdjustPolicy">
                     <enum>QComboBox::AdjustToContents</enum>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="2">
                   <widget class="QLineEdit" name="leFltrFop">
                    <property name="placeholderText">
                     <string>Значення для пошуку</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="3">
                   <widget class="QPushButton" name="pbAddFltrFop">
                    <property name="text">
                     <string>Додати</string>
                    </property>
                    <property name="icon">
                     <iconset resource="res.qrc">
                      <normaloff>:/katynko/svg/draw-cross.svg</normaloff>:/katynko/svg/draw-cross.svg</iconset>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0">
                   <widget class="QToolButton" name="tbRemoveFltrFop">
                    <property name="toolTip">
                     <string>Видалити з пошуку</string>
                    </property>
                    <property name="text">
                     <string>...</string>
                    </property>
                    <property name="icon">
                     <iconset resource="res.qrc">
                      <normaloff>:/katynko/svg/edit-delete.svg</normaloff>:/katynko/svg/edit-delete.svg</iconset>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0" colspan="4">
                   <widget class="QPlainTextEdit" name="pteFltrFop">
                    <property name="readOnly">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>Пошук</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="3" column="1" colspan="2">
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>492</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="0" column="0" colspan="2">
               <widget class="QRadioButton" name="rbAndMode">
                <property name="text">
                 <string>Режим пошуку &quot;І&quot;</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QPushButton" name="pbStartSearch">
                <property name="text">
                 <string>Пошук</string>
                </property>
                <property name="icon">
                 <iconset resource="res.qrc">
                  <normaloff>:/katynko/svg/document-preview-archive.svg</normaloff>:/katynko/svg/document-preview-archive.svg</iconset>
                </property>
               </widget>
              </item>
              <item row="1" column="0" colspan="2">
               <widget class="QRadioButton" name="rbOrMode">
                <property name="text">
                 <string>Режим пошуку &quot;АБО&quot;</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2" rowspan="2">
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>412</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="2" column="0" colspan="3">
               <widget class="QCheckBox" name="cbEnCustomBuffer">
                <property name="text">
                 <string>Буферизація файлу в оперативну пам'ять</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout">
       <item row="4" column="0" colspan="5">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Кожному рядку відповідає інший ключ&lt;/p&gt;&lt;p&gt;Якщо ключ є підмножиною - зробити відступ Tab-ом&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="4">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>466</width>
           <height>21</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="2" colspan="3">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <widget class="QRadioButton" name="rbFop_2">
         <property name="text">
          <string>ФОП</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/katynko/svg/im-user.svg</normaloff>:/katynko/svg/im-user.svg</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pbGoHome">
         <property name="text">
          <string>Домівка</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/katynko/svg/go-home.svg</normaloff>:/katynko/svg/go-home.svg</iconset>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="5">
        <widget class="QStackedWidget" name="swUrOrFop_2">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page_5">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPlainTextEdit" name="pteXmlSchema">
             <property name="plainText">
              <string>urik
</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_6">
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPlainTextEdit" name="pteXmlSchemaFop">
             <property name="plainText">
              <string>fop
</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="rbUr_2">
         <property name="text">
          <string>Юр. особа</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/katynko/svg/im-kick-user.svg</normaloff>:/katynko/svg/im-kick-user.svg</iconset>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="4">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>XML схема документу</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QToolButton" name="tbReset">
         <property name="text">
          <string>Скинути</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/katynko/svg/view-refresh.svg</normaloff>:/katynko/svg/view-refresh.svg</iconset>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
